src/Cron.php

```diff
@@ -106,11 +108,12 @@ public function send_user_has_access_to_post_notification() {
 						if ( $send_email ) {
 							// Woohoo, send the email
 							$userdata = get_userdata( $user_id );
-							if ( ! get_user_meta( $user_id, SIMPLESHOP_PREFIX . 'notification_email_sent_' . $post->ID,
+							$meta_key      = SIMPLESHOP_PREFIX . 'notification_email_sent_' . $post->ID;
```

Comment:
Velká mezera za `$meta_key`

Reply:
Je to dle: https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/